Microsoft.Toolkit.Win32.UI.XamlApplication.Package/Microsoft.Toolkit.Win32.UI.XamlApplication.Package.csproj

```diff
@@ -61,6 +61,23 @@
       <PackagePath>runtimes\win10-x64\native</PackagePath>
     </Content>
 
+    <Content Include="..\arm\$(Configuration)\Microsoft.Toolkit.Win32.UI.XamlApplication\Microsoft.Toolkit.Win32.UI.XamlHost.dll">
+      <Pack>true</Pack>
+      <PackagePath>runtimes\win10-arm\native</PackagePath>
+    </Content>
+
+    <!--
```

Comment:
I believe the PDB issue has been resolved as we are flattening WAP projects, so this no longer required and we can safely ship the PDBs

Reply:
Let's fix this on another PR.

Reply:
I'm curious about what this means exactly. Are all WAP projects now flattened, meaning the previous workaround of adding the following target to the `.wapproj` is now not needed anymore?
```xml
<Target Name="FlattenPackage" BeforeTargets="_ConvertItems">
  <ItemGroup>
    <_TemporaryFilteredWapProjOutput Include="@(_FilteredNonWapProjProjectOutput)" />
    <_FilteredNonWapProjProjectOutput Remove="@(_TemporaryFilteredWapProjOutput)" />
    <_FilteredNonWapProjProjectOutput Include="@(_TemporaryFilteredWapProjOutput)">
      <SourceProject></SourceProject>
    </_FilteredNonWapProjProjectOutput>
  </ItemGroup>
</Target>
```